mantidproject/mantidimaging
# Add coverage and upload to Coveralls to Actions

Fixes #712 

To Test:
Check that the build from this PR appears here:
https://coveralls.io/github/mantidproject/mantidimaging

We will want to watch this after it is merged to ensure master is being built correctly as well.
